recipes/adjfind/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "adjfind" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zwang-geog/AdjFind/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 70411994bb4f1a6f97dcf4e6568b0df08cf20e67aa1e8d6c7fa45a9c80cb02a6
+
+build:
+  number: 0
+  script: |
+    mkdir build
+    cd build
+    cmake ${CMAKE_ARGS} ..  # [unix]
+    cmake %CMAKE_ARGS% ..  # [win]
+    cmake --build . --config Release
+    cmake --install .
+
+requirements:
+  build:
+    - cmake
+    - make  # [unix]
+    - git
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+  host:
+    - libgdal-core
+    - libboost-devel
+  run:
+    - libgdal-core
+    - libboost
```

Comment:
This should be handled by the `run_exports` automatically, there is any need to manually add them?